sdk/cosmos/azure-cosmos-spark_3-3_2-12/CHANGELOG.md

```diff
@@ -0,0 +1,296 @@
+## Release History
+
+### 4.15.0-beta.1 (Unreleased)
+
+#### Features Added
+
+#### Breaking Changes
+
+#### Bugs Fixed
+
+#### Other Changes
+
+### NOTE: Below versions only exist for Spark 3.2 -
```

Comment:
it exists for 3.1 and 3.2. Instead of the note, should we just start from fresh?